Test failure:
```
11:23:33 [ERROR] Tests run: 20, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 2.765 s <<< FAILURE! - in com.yahoo.vespa.hosted.controller.deployment.InternalStepRunnerTest
11:23:33 [ERROR] realDeploymentRequiresForTesterCert  Time elapsed: 0.755 s  <<< FAILURE!
11:23:33 org.opentest4j.AssertionFailedError: expected: <succeeded> but was: <unfinished>
```

--- a/controller-server/src/test/java/com/yahoo/vespa/hosted/controller/deployment/InternalStepRunnerTest.java
+++ b/controller-server/src/test/java/com/yahoo/vespa/hosted/controller/deployment/InternalStepRunnerTest.java
@@ -516,7 +516,7 @@ public class InternalStepRunnerTest {
         assertEquals(oldTrusted, tester.configServer().application(app.instanceId(), id.type().zone()).get().applicationPackage().trustedCertificates());
 
         tester.configServer().throwOnNextPrepare(null);
-        tester.clock().advance(Duration.ofSeconds(300));
+        tester.clock().advance(Duration.ofSeconds(450));
         tester.runner().run();
         assertEquals(succeeded, tester.jobs().run(id).stepStatuses().get(Step.deployTester));
         assertEquals(succeeded, tester.jobs().run(id).stepStatuses().get(Step.deployReal));

